docs/concepts/editions-overview.md

```diff
@@ -0,0 +1,51 @@
+
+There are multiple applicatons stacks built on top the .NET Framework that allow developers to build applications ranging from console applications, across rich client (WPF) applications to scalable web applications.
+
+[Windows Forms](https://msdn.microsoft.com/en-us/library/dd30h2yb%28v=vs.110%29.aspx) and [Windows Presentation Foundation (WPF)](https://msdn.microsoft.com/en-us/library/ms754130%28v=vs.110%29.aspx) are User Interface (UI) stacks that enable you to build desktop applications for Windows. The strength of Windows Forms lies in its rich support for common databinding scenarios as well as access to Windows’ native user interface controls. WPF, on the other hand, allows you to exercise much more control over the look and feel of your application. 
+
+[Windows Communication Foundation (WCF)](https://msdn.microsoft.com/en-us/library/ms731082%28v=vs.110%29.aspx) is a set of libraries for SOAP Web Services. It allows you to create services that can communicate through various supported protocols using various data formats, and that can be hosted in any process you choose. This leads to one of the major features of WCF: your services are not tied to any particular hosting strategy or approach.
+
+[ASP.NET](http://www.asp.net/) is a web framework. Being a very rich framework, it has several distinct pieces which are used to produce modern and high-performance web applications. [ASP.NET Web Forms](http://www.asp.net/web-forms) is a set of tools geared primarily towards developer productivity, allowing quick turnaround on web applications with a drag-and-drop surface reusing web controls for everything from logging to data binding. [ASP.NET MVC](http://www.asp.net/mvc)gives you greater control over the entire web pipeline, from the HTTP layer to the user interface. [ASP.NET WebAPI](http://www.asp.net/web-api) is a convention-based framework for creating REST services. It allows you to set-up a REST endpoint extremely fast. Finally, [SignalR](http://www.asp.net/signalr) allows you to provide push-based communication to your web applications using the [WebSocket](https://en.wikipedia.org/wiki/WebSocket) protocol.
+
+## Mono
+
+[Mono](http://www.mono-project.com/docs/about-mono/) is the original open source and cross-platform implementation of .NET, from the community [Mono Project](http://mono-project.com). It is now sponsored by [Xamarin](http://xamarin.com) It can be thought of as an open and cross-platform version of the .NET Framework. It's APIs follow the progress of the .NET Framework, not .NET Core.
+
+There are several components that make up Mono:
+
+**C# Compiler** - Mono’s C# compiler is feature complete for C# 1.0, 2.0, 3.0, 4.0 and 5.0 (ECMA). A good description of the feature of the various versions is available on Wikipedia.
```

Comment:
Funny. That content came from: http://www.mono-project.com/docs/about-mono/. Fixed.